### PR TITLE
blog: meet our contributors APAC China region

### PR DESCRIPTION
**Pull request by jayesh-srivastava**
This PR adds the third blog post for China region, in the series of ["Meet Our Contributors - APAC"](https://docs.google.com/document/d/1zB-FOvfbnKbiWlGgInnDB06hDfLsTQuuKsXQCqc-lL0/edit#heading=h.yr3qa2ko7xq) blog series.

Reference issue: https://github.com/kubernetes/community/issues/4469#issuecomment-942422348

### CODE REVIEW

File: content/en/blog/2022/meet-our-contributors-APAC-China-region-03.md
Comment (jberkus) [inline content/en/blog/2022/meet-our-contributors-APAC-China-region-03.md:27]:
```suggestion
His open source contributions are mainly self-motivated. In the last two years he has mentored a few students contributing to Kubernetes through the LFX Mentorship program, some of whom got jobs due to their expertise and contributions on Kubernetes projects. 
```

**Review by jberkus — CHANGES_REQUESTED** [review]
Looks good!  Just some minor editorial corrections.

**Comment by jberkus** [comment]
Also ... are you sure you want to publish this on 6/11, a Saturday?

File: content/en/blog/2022/meet-our-contributors-APAC-China-region-03.md
Comment (jayesh-srivastava) [reply content/en/blog/2022/meet-our-contributors-APAC-China-region-03.md:12]: Yeah makes sense. I'll change this to a simple 'Hello'.

File: content/en/blog/2022/meet-our-contributors-APAC-China-region-03.md
Comment (jayesh-srivastava) [reply content/en/blog/2022/meet-our-contributors-APAC-China-region-03.md:62]: Maybe following them on socials and trying to understand their projects and stuff. I don't really know what this means.

File: content/en/blog/2022/meet-our-contributors-APAC-China-region-03.md
Comment (jayesh-srivastava) [reply content/en/blog/2022/meet-our-contributors-APAC-China-region-03.md:38]: This was the response to, 'Have you worked with Kubernetes before?'. IMO, we can keep this, after some minor adjustments.
`As of his prior experience, Shiming had been involved with Kubernetes at his company`

**Comment by jayesh-srivastava** [comment]
> Also ... are you sure you want to publish this on 6/11, a Saturday?

Didn't think about this. Will change this to the 13th of June.

**Comment by jayesh-srivastava** [comment]
@jberkus Can you review this now please?

File: content/en/blog/2022/meet-our-contributors-APAC-China-region-03.md
Comment (jberkus) [reply content/en/blog/2022/meet-our-contributors-APAC-China-region-03.md]: Still doesn't parse, sorry.  You've already mentioned his employer; what does this sentence add, at all, to the story?

File: content/en/blog/2022/meet-our-contributors-APAC-China-region-03.md
Comment (jberkus) [reply content/en/blog/2022/meet-our-contributors-APAC-China-region-03.md:63]: Need to figure that out, then, before we can publish.

File: content/en/blog/2022/meet-our-contributors-APAC-China-region-03.md
Comment (jberkus) [reply content/en/blog/2022/meet-our-contributors-APAC-China-region-03.md]: (specifically, "as of his prior experience" is not grammatical, to the point where it's not clear what meaning is intended.  I'd replace it with something else, but as I say above, you've already mentioned his company, so the whole sentence feels repetitive)

**Comment by jberkus** [comment]
Jayesh,

There's still some of the issues I called out in my first review.

File: content/en/blog/2022/meet-our-contributors-APAC-China-region-03.md
Comment (jayesh-srivastava) [reply content/en/blog/2022/meet-our-contributors-APAC-China-region-03.md]: > Still doesn't parse, sorry. You've already mentioned his employer; what does this sentence add, at all, to the story?
Besides mentioning the employer, we have to talk about how the contributor started with Kubernetes/was involved with Kubernetes. Hence the sentence. If you can suggest any other way of including the sentence that would be great.

**Comment by sftim** [comment]
If we want this mirrored onto https://k8s.io/blog/ (which is usual), please open a PR there once the final contents are agreed.

I suggest delaying a day or two to get that sorted. We can /hold this merge, get approvals, and then unhold it on publication day (cf https://github.com/kubernetes/contributor-site/issues/281)

File: content/en/blog/2022/meet-our-contributors-APAC-China-region-03.md
Comment (sftim) [inline content/en/blog/2022/meet-our-contributors-APAC-China-region-03.md:61]:
```suggestion
Currently Jintao is a reviewer for the [ingress-nginx](https://kubernetes.github.io/ingress-nginx/) project.
```

File: content/en/blog/2022/meet-our-contributors-APAC-China-region-03.md
Comment (sftim) [inline content/en/blog/2022/meet-our-contributors-APAC-China-region-03.md:49]:
```suggestion
Currently, Paco is a reviewer for `kubeadm` (a SIG Cluster Lifecycle product), and for SIG Node.
He states that the maintainers have helped him a lot in learning and contributing to the Kubernetes project.
```
?

File: content/en/blog/2022/meet-our-contributors-APAC-China-region-03.md
Comment (sftim) [inline content/en/blog/2022/meet-our-contributors-APAC-China-region-03.md:36]:
```suggestion
He has mostly been involved with SIG Node, as a reviewer.
```

**Review by sftim — COMMENTED** [review]
Some wording suggestions

**Comment by jayesh-srivastava** [comment]
> I suggest delaying a day or two to get that sorted. We can /hold this merge, get approvals, and then unhold it on publication day (cf https://github.com/kubernetes/contributor-site/issues/281)

@sftim I have updated the date too.

**Comment by jayesh-srivastava** [comment]
@sftim @jberkus  Are there more changes to be made? The date has to be changed tho.

File: content/en/blog/2022/meet-our-contributors-APAC-China-region-03.md
Comment (jberkus) [inline content/en/blog/2022/meet-our-contributors-APAC-China-region-03.md:38]:
```suggestion
Shiming Zhang is a Software Engineer working on Kubernetes for DaoCloud in Shanghai, China.

He has mostly been involved with SIG Node, as a reviewer. His major contributions have mainly been bug fixes and feature improvements in an ongoing [KEP](https://github.com/kubernetes/enhancements/tree/master/keps/sig-node/2712-pod-priority-based-graceful-node-shutdown), all revolving around SIG Node.
```

Comment (jberkus) [reply content/en/blog/2022/meet-our-contributors-APAC-China-region-03.md:38]: This is my suggested re-write to eliminate the problematic sentence.

File: content/en/blog/2022/meet-our-contributors-APAC-China-region-03.md
Comment (jberkus) [reply content/en/blog/2022/meet-our-contributors-APAC-China-region-03.md:63]: This still needs to either be fixed, or cut.  Thanks!

File: content/en/blog/2022/meet-our-contributors-APAC-China-region-03.md
Comment (jayesh-srivastava) [reply content/en/blog/2022/meet-our-contributors-APAC-China-region-03.md:38]: @jberkus Done!

Comment (jayesh-srivastava) [reply content/en/blog/2022/meet-our-contributors-APAC-China-region-03.md:38]: @jberkus Done!

File: content/en/blog/2022/meet-our-contributors-APAC-China-region-03.md
Comment (jayesh-srivastava) [reply content/en/blog/2022/meet-our-contributors-APAC-China-region-03.md:63]: @jberkus Do you have any suggestions for this as well?

Comment (jberkus) [reply content/en/blog/2022/meet-our-contributors-APAC-China-region-03.md:63]: No, because I have no idea what was meant by it.  Did you query Jintao?

File: content/en/blog/2022/meet-our-contributors-APAC-China-region-03.md
Comment (jayesh-srivastava) [reply content/en/blog/2022/meet-our-contributors-APAC-China-region-03.md]: @jberkus So I pinged Jintao, for more context to this statement. This is his reply.
`At present, there are many open-source commercialization companies.
For example, I am working on [API7.ai](http://api7.ai/), of course there are many well known companies like HashiCorp, MongoDB, Docker Inc. etc. In these companies, full-time open source positions are often offered. Engineers can focus on open source projects and communities. And for these companies, open source projects and communities are the cornerstone of their success. So they will be more willing to invest in this`

How should we accommodate this into the blog response?

File: content/en/blog/2022/meet-our-contributors-APAC-China-region-03.md
Comment (jberkus) [reply content/en/blog/2022/meet-our-contributors-APAC-China-region-03.md:63]: Hey, this feedback is still unresolved, which is why we have not published.

Comment (jayesh-srivastava) [reply content/en/blog/2022/meet-our-contributors-APAC-China-region-03.md:63]: @jberkus I don't know how but my comment got deleted which consisted of Jintao's response to this.

So I pinged him regarding this to provide more context to his statement. He says
`At present, there are many open source commercialization companies.
For example I am working on [API7.ai](http://api7.ai/), of course there are many well known companies like HashiCorp, MongoDB, Docker Inc. etc. In these companies, full-time open source positions are often offered. Engineers can focus on open source projects and communities. And for these companies, open source projects and communities are the cornerstone of their success. So they will be more willing to invest in this`

How should we accommodate this into the blog?

Comment (jberkus) [reply content/en/blog/2022/meet-our-contributors-APAC-China-region-03.md:63]:
```suggestion
He suggests keeping track of job opportunities at open source companies so that you can get a job contributing full time. For new contributors Jintao says that if anyone wants to make a significant contribution to an open source project, then they should choose the project based on their interests and should generously invest time.
```

**Comment by jayesh-srivastava** [comment]
@jberkus Have made the changes.

**Comment by jayesh-srivastava** [comment]
@jberkus @sftim I guess this can be merged now. I changed the publish date to today's only. Also the hold label needs to be removed. I am not able to do it.

**Review by Debanitrkl — APPROVED** [review]
/lgtm

**Comment by jberkus** [comment]
@Debanitrkl @kaslin @sftim we need a date for this, since it'll go on the Kubernetes blog as well.

**Comment by jayesh-srivastava** [comment]
@Debanitrkl @kaslin @sftim @jberkus Have we decided on the date? Should I change it in the blog?

**Comment by sftim** [comment]
11th would be fine by me. Is there a PR against https://github.com/kubernetes/website already?

/hold
We can remove the hold on the publication date.

**Comment by jayesh-srivastava** [comment]
@sftim No there isn't a PR for k/website. I'll make it now as we have decided on the date. Also, I've updated the date for this PR.

**Comment by jayesh-srivastava** [comment]
@sftim This is the PR against k/website. (https://github.com/kubernetes/website/pull/34865)

**Comment by jayesh-srivastava** [comment]
@sftim @Debanitrkl @jberkus Since there are more reviews on the PR against the k/website which haven't been finalized yet, I think we can hold this PR too until the 18th which is a Monday.

**Comment by sftim** [comment]
@jayesh-srivastava I'm a reviewer for both blogs; I'm fine with the idea of taking the text from here as the final version. It's nice to incorporate any feedback from SIG Docs' blog team, but not _essential_.

File: content/en/blog/2022/meet-our-contributors-APAC-China-region-03.md
Comment (sftim) [inline content/en/blog/2022/meet-our-contributors-APAC-China-region-03.md:4]: How about we update this article (and its mirror) to publish on August 15th?

Comment (jayesh-srivastava) [reply content/en/blog/2022/meet-our-contributors-APAC-China-region-03.md:4]: @sftim Sure. Some suggestions on the k/website PR are still not resolved, so I was waiting for the SIG Docs members to comment on those but it's been weeks now. I'll make some minor changes and prepare the article and the mirror to get published on August 15th now.

**Comment by jayesh-srivastava** [comment]
@sftim I've updated this article as well as the mirror (https://github.com/kubernetes/website/pull/34865) to get published on August 15th. Can you please review them?

File: content/en/blog/2022/meet-our-contributors-APAC-China-region-03.md
Comment (sftim) [inline content/en/blog/2022/meet-our-contributors-APAC-China-region-03.md:6]:
```suggestion
```
You set a `canonicalUrl` on the **mirror** of the article, and you don't set this field on the original.

**Review by sftim — COMMENTED** [review]
Just one thing to fix.

**Comment by sftim** [comment]
/label tide/merge-method-squash
/hold
until publication date - see https://github.com/kubernetes/contributor-site/issues/281

/lgtm
/approve

**Comment by sftim** [comment]
/hold cancel

Ready to publish

**Pull request merged**
